### PR TITLE
Feat/one page per question

### PR DESCRIPTION
**Pull request by carolinwagner**
Please review :-) 

Preview: https://capstone-project-git-feat-one-page-per-question.carolinwagner.vercel.app

Closes #15

### CODE REVIEW

**Review by jennydely — COMMENTED** [review]
Looks good,
find some comments in the code.

Sometimes it feeld, like you using divs into divs, which are not neccessary. Maybe you can prove this? Sometimes it could be possible to use global styles for main, form etc. instead of so many different divs?

File: src/App.js [hunk 1]
```diff
@@ -4,30 +4,47 @@ import Header from './Header'
...
+          </Route>
+          <Route path="/questions">
+            <QuestionsForm onAddAnswer={addAnswer} />
+          </Route>
+          <Route path="/bylawstext">
+            <BylawsText answers={answers} />
+          </Route>
+        </Switch>
+      </StyledMainContainer>
+    </StyledCenterOnDesktop>
```
[the remark this answers is not in the document]
Comment (carolinwagner) [reply src/App.js:28]: Yes, it is coded for mobile first, but I also wanted it to look nice on desktops.

File: src/BylawsText.js [hunk 2]
```diff
@@ -27,12 +27,17 @@ export default function BylawsText({ answers }) {
   })
   const LocationAndDate = (
     <h4 id="locationAndDate">
-      {answers.clubLocation}, {answers.decisionDate}
```
[the remark this answers is not in the document]
Comment (carolinwagner) [reply src/BylawsText.js:30]: I am using the ids for cypress tests - could also substitute them with data-cys though. 
The ids are all unique.

File: src/App.js [hunk 3]
```diff
@@ -4,30 +4,47 @@ import Header from './Header'
...
+  width: 100%;
+`
+const StyledCenterOnDesktop = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+`
+
 export default App
```
[the remark this answers is not in the document]
Comment (carolinwagner) [reply src/App.js:50]: I am using these divs for styling - one div is a flex box with flex-direction column, the other flex box has the direction rows.

**Pull request merged**